### DISCUSSION TITLE
Multi step form on one router

### DISCUSSION THREAD

**Discussion by nani1008** [react-hook-form/react-hook-form#6611] [Ideas]
I have a multi step form with 1-5 radio buttons on each form. Each form has no continue button to proceed onto the next form. As of now I can go to the next form using `onChange` inside a `<div>` but when I return to the previous form I cant pass thru it if I selected the same radio button. How can I go to the next form without using any button and get still the value from the the fields?

**Comment by bluebill1049** [comment]
This is a router question insted of form, I would suggest to keep each page a separate form and use router library to navigate.

**Reply by nani1008** [reply]
What Im trying to do is a questionaire and the questions are not static so separate form is not a thing. Any other ideas?

**Reply by bluebill1049** [reply]
https://react-hook-form.com/advanced-usage#WizardFormFunnel ?
